Makefile.am

```diff
@@ -128,7 +128,11 @@ EXTRA_DIST = test/fixtures/empty_file \
 
 TESTS = test/run-tests
 check_PROGRAMS = test/run-tests
+if OS390
+test_run_tests_CFLAGS =
```

Comment:
the C compiler on zOS by default does not issue warnings for long-long type being used.